### DISCUSSION TITLE
Keycloak as *only* a login provider?

### DISCUSSION THREAD

**Discussion by claym** [keycloak/keycloak#32069] [Q&A]
Is it feasible to use Keycloak as simply a login provider? My use-case has a different provider for both human and machine tokens, so I simply need Keycloak to simply handle logins, via password or social media/oidc.

In this case Keycloak would only be needed to authenticate the user and then provide a session cookie or similar mechanism to verify identity.

Can Keyclock operate in this situation, or am I barking up the wrong tree here?

**Comment by darius-m** [comment]
Can you elaborate about what would differ from normal authentication flows? What you've described sounds like a regular SSO, which Keycloak is / can be.
